package.json

```diff
@@ -25,9 +25,10 @@
     },
     "devDependencies": {
         "cy-verify-downloads": "^0.1.13",
-        "cypress": "^6.9.1",
+        "cypress": "^12.11.0",
```

Comment:
Testing library version needs to be updated

"@testing-library/cypress": "^9.0.0",

Reply:
Regarding node version - didn't notice that cause I'm running 14. The library - I will update it.